pkg/kubelet/pleg/generic.go

```diff
@@ -138,4 +147,7 @@ func (g *GenericPLEG) relist() {
 	for i := range events {
 		g.eventChannel <- events[i]
 	}
+
+	// Update the relist time.
+	g.lastRelistTime = timestamp
```

Comment:
Maybe move this up after `metrics.PLEGRelistInterval.Observe()` or in the defer clause ?